manifests/puppetdb_exporter.pp

```diff
@@ -55,7 +55,7 @@
   String[1] $package_ensure               = 'present',
   String[1] $package_name                 = 'puppetdb_exporter',
   String[1] $user                         = 'puppetdb-exporter',
-  String[1] $version                      = '1.0.0',
+  String[1] $version                      = '1.1.0',
```

Comment:
I think changing the default version installed is a breaking change?  Perhaps a several default exporter versions could be batched into a single release?

Reply:
yes breaking change. But I would like to update the other exporters on individual PRs (and we already have a few other backwards-incompatible changes on master)